standard/date_ext.lua

```diff
@@ -57,27 +57,25 @@ end
 --- The format string is the same used by mw.language.formatDate and {{#time}}.
 ---@param format string
 ---@param timestamp string|integer
----@return string
+---@return string|number
 function DateExt.formatTimestamp(format, timestamp)
 	return mw.getContentLanguage():formatDate(format, '@' .. timestamp)
 end
 
 --- Converts a date string or timestamp into a format that can be used in the date param to Module:Countdown.
----@param format string
 ---@param dateOrTimestamp string|integer
 ---@return string
 function DateExt.toCountdownArg(dateOrTimestamp)
 	local timestamp = DateExt.readTimestamp(dateOrTimestamp)
-	return DateExt.formatTimestamp('F j, Y - H:i', timestamp) .. ' <abbr data-tz="+0:00"></abbr>'
+	return DateExt.formatTimestamp('F j, Y - H:i', timestamp or '') .. ' <abbr data-tz="+0:00"></abbr>'
```

Comment:
This `or ''` is in the wrong place no?

Reply:
`formatTimestamp` always returns a string or number
but it can not read nil (would be nil concat due to `'@' .. timestamp` in line 62)

Reply:
Ah right, misread it as calling readTimestamp